content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -191,7 +211,18 @@ describe("GIVEN setValues function", () => {
             <input type="text" id="driver-pod-label" value="dell-storage">
             <input type="checkbox" id="volumeless-pods">
             <input type="checkbox" id="connection-validation">
+            <input type="text" id="target-cluster-id" value="">
+            <input type="text" id="transport-protocol" value="">
+            <input type="checkbox" id="iscsichap">
+            <input type="checkbox" id="observability-operator">
+            <input type="checkbox" id="observability-operator-metrics">
+            <input type="checkbox" id="vobservability-operator-topology">
```

Comment:
There is a typo in the id "vobservability-operator-topology

Reply:
fixed .